ggshield/core/constants.py

```diff
@@ -3,6 +3,8 @@
 
 # max file size to accept
 MAX_FILE_SIZE = 1048576
+# max files size to create a tar from
+MAX_TOTAL_FILES_SIZE = 31457280  # 30MB
```

Comment:
Naming: `MAX_TAR_CONTENT_SIZE` would be more self-explanatory.
Also: instead of writing ` = 31457280  # 30MB` you can write ` = 30 * 1024 * 1024`